test/quoteSpec.js

```diff
@@ -266,7 +266,7 @@ describe('Quotes', function () {
         destination_ledger: 'usd-ledger.',
         destination_amount: '1056024', // EUR/USD Rate of 1.0592 - .2% spread - slippage
         destination_expiry_duration: '5',
-        liquidity_curve: [ [1000, 0], [1000000000000, 1057081598942.9185] ]
+        liquidity_curve: [ [999.9999999999999, 0], [1000000000000, 1057081598942.9186] ]
```

Comment:
these values changed slightly because I used numbers, not BigNumbers, to apply currency exchange rates